src/main/scala/com/cognite/sdk/scala/v1/dataModels.scala

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+package com.cognite.sdk.scala.v1
+
+import io.circe.Json
+
+final case class DataModelProperty(
+    `type`: String,
+    nullable: Option[Boolean] = Some(true)
+)
+
+final case class DataModelPropertyIndex(
+    indexName: Option[String] = None,
+    fields: Option[Seq[String]] = None
+)
+
+final case class DataModel(
+    externalId: String,
+    properties: Option[Map[String, DataModelProperty]] = None,
+    `extends`: Option[Seq[String]] = None,
```

Comment:
it's optional on the api side so I prefer to keep the same, it might change the behavior if we send an empty array "extend":[]  instead of null or non represent field 😅

Reply:
True! Is that what happens, though? I mean, does circe actually send that, or is the field removed if the Seq is empty?

Reply:
I just list all possible scenarios below so you can have a clear view of it hehe 😁 
If the the field is not optional, circe will send `"extends": []` for empty array
If the field is optional and 

- extends is `None` and we use implicit `dropNullValue = true` => field is not sent at all, if not circe send `"extends": null`
- extends is `Some(...)`, circe just sends `"extends":[...]`

Reply:
Awesome, thanks for the explanation!